### PR TITLE
Hover button, font size improvement

### PR DESCRIPTION
**Pull request by ojasaklechayt**
Fixes: #210 

### What's the style idea?

Details:
1. Text Size Deceased.
2. Navbar Names Hover Improvement.
3. Content added for " What is DevsInTech Community? ".

### Add screenshots

Before:
image
image

After:
image
image

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Add screenshots

Add screenshots

### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

### CODE REVIEW

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey @ojasaklechayt there are some unwanted spaces between text kindly check fix them 

Screenshot from 2023-06-08 11-14-57

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey @ojasaklechayt there are some unwanted spaces between text kindly check fix them 

Screenshot from 2023-06-08 11-14-57

**Comment by ojasaklechayt** [comment]
> Hey @ojasaklechayt there are some unwanted spaces between text kindly check fix them
> 
> Screenshot from 2023-06-08 11-14-57

okay ser I will look into it

**Comment by ojasaklechayt** [comment]
> > Hey @ojasaklechayt there are some unwanted spaces between text kindly check fix them
> > Screenshot from 2023-06-08 11-14-57
> 
> okay ser I will look into it

done ser

**Comment by ojasaklechayt** [comment]
@Susmita-Dey @himanshu1221 any updates on this ?

**Comment by ojasaklechayt** [comment]
@Susmita-Dey @himanshu1221 ?

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Hey @ojasaklechayt Kindly fix the merge conflict and I can still see some unwanted spaces in the above ss

File: src/utils/helper/validation.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+export const EMAIL_REGEX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+
+export const Name_REGEX = /^[A-Za-z ]+$/;
+
+export const PASSWORD_REGEX =
+  /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$/;
+
+export const TITLE_REGEX = /^(?!\s*$).+/;
+
+export const Mobile_REGEX = /^[0-9]{10}$/;
```
Comment (Susmita-Dey) [inline src/utils/helper/validation.js:10]: What's the usecase of this file?

Comment (ojasaklechayt) [reply src/utils/helper/validation.js:10]: I feel this is for chatbot. It's not my code

**Comment by Susmita-Dey** [comment]
Hey @ojasaklechayt Ping me when this PR is ready to merge(you're done with the tasks)

**Comment by Susmita-Dey** [comment]
> Shall I remove this validation.js code ?
Yes. I don't see it's use case. Also, it's don't related to your issue/PR

**Comment by ojasaklechayt** [comment]
@Susmita-Dey I feel PR is ready. Can you please check once ?

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
image

Make the text short in footer as much as possible

**Comment by Susmita-Dey** [comment]
Hey @ojasaklechayt Also, don't forget to resolve merge conflicts

**Comment by ojasaklechayt** [comment]
@Susmita-Dey  Resolved the conflicts and made the text shorter

**Review by Susmita-Dey — APPROVED** [review]
LGTM

**Pull request merged**